### PR TITLE
Add Date Helper Functions

### PR DESCRIPTION
**Pull request by RamVasuthevan**

### CODE REVIEW

File: website/script.js [hunk 1]
```diff
@@ -1,33 +1,34 @@
+function createDate(year, month, day, time = null) {
+    // Creates a moment object with the specified date and time (optional) in America/Toronto timezone
+    const dateStr = `${year}-${month}-${day}` + (time ? ` ${time}` : '');
+    return moment.tz(dateStr, time ? 'YYYY-MM-DD HH:mm' : 'YYYY-MM-DD', 'America/Toronto');
+}
+
+function createToday(){
+    return moment.tz('America/Toronto')
+}
```
Comment (kyleyarwood) [inline website/script.js:9]: nit: we could take in the timezone as an argument as well which would default to America/Toronto

**Review by kyleyarwood — APPROVED** [review]
LGTM, just one nit that you can ignore if you want. I don't feel strongly considering we should never be using another timezone for Toronto ferries

File: website/script.js (hunk 1 above)
Comment (RamVasuthevan) [reply website/script.js:9]: Hmm ... I  don't think we'll ever need it, and I think that `createDate` and `createToday` are modular enough that doing it in the future would be pretty easy.

Comment (RamVasuthevan) [reply website/script.js:9]: I appreciate your detail oriented comment

**Pull request merged**